libs/checkout/state/src/checkout-state.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+
+import { DaffCartStateModule } from '@daffodil/cart/state';
+import { DaffOrderStateModule } from '@daffodil/order/state';
+
+
+@NgModule({
+  imports: [
+    DaffCartStateModule,
+    DaffOrderStateModule,
```

Comment:
I assume you mean as we add features to checkout. We could piece up the modules per feature, i.e. this would be the `DaffCheckoutPlacedOrderStateModule`